backend/core/src/migration/1668556487187-add-home-types.ts

```diff
@@ -0,0 +1,14 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class addHomeTypes1668556487187 implements MigrationInterface {
```

Comment:
I would have anticipated seeing a type created here for the home_type instead of `character varying` as happened here: https://github.com/CityOfDetroit/bloom/blob/dev/backend/core/src/migration/1649374032458-marketing-season.ts

Was this migration auto generated?